### PR TITLE
Core RP - preview camera background color

### PR DESCRIPTION
**Pull request by martint-unity**
---
### Purpose of this PR
Adding functionality to adjust the background color of the preview camera.
Settings can be found in Preferences / Colors / SRP

preview_color_backgrounds
---
### Testing status

Running Yamato

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.render-pipelines.core/Runtime/Utilities/CoreRenderPipelinePreferences.cs [hunk 1]
```diff
@@ -27,26 +35,56 @@ public static Color volumeGizmoColor
             }
         }
 
+        public static Color previewBackgroundColor
```
Comment (phi-lira) [inline com.unity.render-pipelines.core/Runtime/Utilities/CoreRenderPipelinePreferences.cs:38]: Add API docs to any new public API.

**Comment by martint-unity** [comment]
@TomasKiniulis 
All of your tests are reproducible in Trunk and Master without my changes. Please confirm.

**Comment by phi-lira** [comment]
Fix merge conflicts and drop me a message so I merge this one.

**Comment by martint-unity** [comment]
There are some issues with not updating background after created Prefab or Material. Will have to investigate.

**Pull request closed without merging**